src/components/Typography/index.tsx

```diff
@@ -280,17 +282,24 @@ const TextWithTooltip = ({ children, ...props }: TypographyProps) => {
 
   return (
     <Tooltip text={finalStringChildren}>
-      {({ onMouseEnter = () => {}, onFocus = () => {}, ...tooltipProps }) => (
+      {({
+        onMouseEnter = () => {},
+        onFocus = () => {},
+        ...tooltipProps
+      }: {
+        onMouseEnter: EventHandler<MouseEvent>
+        onFocus: EventHandler<FocusEvent>
```

Comment:
can't we use `MouseEvent<HTMLButtonElement>` for `onMouseEnter` instead ?
and `FocusEvent<HTMLInputElement>` for `onFocus` ?

Reply:
You're absolutely right, done :)